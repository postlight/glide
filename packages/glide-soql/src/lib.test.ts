import soql, { from } from "./lib";

test("soql()", () => {
  expect(soql("User")).toMatchSnapshot();
  expect(soql(from("User"))).toMatchSnapshot();
});
