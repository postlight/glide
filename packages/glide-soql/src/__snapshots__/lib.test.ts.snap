// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`soql() 1`] = `
Builder {
  "parts": Object {
    "groupBy": GroupBy {
      "fields": Array [],
      "having": undefined,
    },
    "limit": Limit {
      "size": undefined,
    },
    "offset": Offset {
      "size": undefined,
    },
    "orderBy": OrderBy {
      "direction": "ASC",
      "fields": Array [],
    },
    "select": Select {
      "fields": Set {},
      "object": "User",
    },
    "where": Where {
      "condition": undefined,
    },
  },
}
`;

exports[`soql() 2`] = `
Builder {
  "parts": Object {
    "groupBy": GroupBy {
      "fields": Array [],
      "having": undefined,
    },
    "limit": Limit {
      "size": undefined,
    },
    "offset": Offset {
      "size": undefined,
    },
    "orderBy": OrderBy {
      "direction": "ASC",
      "fields": Array [],
    },
    "select": Select {
      "fields": Set {},
      "object": "User",
    },
    "where": Where {
      "condition": undefined,
    },
  },
}
`;
